src/license/mapping.xml

```diff
@@ -42,6 +42,31 @@
         <artifactId>org.osgi.core</artifactId>
         <license>Apache License, Version 2.0</license>
     </artifact>
+    <artifact>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcmail-jdk15on</artifactId>
+        <license>Bouncy Castle/MIT License</license>
+    </artifact>
+    <artifact>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
these ones didn't get automatically picked up either? let me know and I'll open an issue against BC (this info _should_ bet getting pulled from the project pom file)